sensu/plugins/check-glance-store.py

```diff
@@ -15,6 +15,11 @@
 from iso8601 import iso8601
 from keystoneclient.v2_0 import client as ksclient
 
+def is_remote_image(image):
```

Comment:
i don't think this is possible.  I found that some images had location set to remote but they still had an image stored.
So I took the "better, but not optimal" path of allowing images that do have a remote location set to either have a file in the backing store or not.